src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -99,13 +103,15 @@ public PostmanImporter(TestCreator testCreator) {
     public WsdlProject importPostmanCollection(WorkspaceImpl workspace, String filePath) {
         WsdlProject project = null;
         String postmanJson = null;
-        UrlClientLoader loader = new UrlClientLoader(filePath);
-        loader.setUseWorker(false);
+        XProgressDialog collectionImportProgressDialog = UISupport.getDialogs().createProgressDialog("Import Collection",
+                0, "Importing a collection", false);
```

Comment:
I suggest the following: ``Importing the collection...``

Reply:
@Alexander-Sviridov, ok